ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ntp.j2

```diff
@@ -1,16 +1,61 @@
 {# eos - NTP #}
-{% if ntp.authenticate is arista.avd.defined(true) %}
+{% if ntp is arista.avd.defined %}
+!
+{%     for local_interface in ntp.local_interfaces | arista.avd.natural_sort('name') %}
```

Comment:
As mentioned, only one interface is allowed in EOS

Reply:
Ack. Fix on it's way.